CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased] - 2019-07-12
+
```

Comment:
Also, the format of the change log (see https://keepachangelog.com/en/1.0.0) requires to classify the changes by category, in this case these qualify as addtiions.

```suggestion

### Added

```